alacritty/src/input.rs

```diff
@@ -492,12 +493,8 @@ impl<'a, T: EventListener, A: ActionContext<T> + 'a> Processor<'a, T, A> {
             };
             self.mouse_report(code, ElementState::Released, modifiers);
             return;
-        } else if let (Some(point), true) = (point, button == MouseButton::Left) {
-            let mouse_state = self.mouse_state(point, modifiers);
-            self.update_mouse_cursor(mouse_state);
-            if let MouseState::Url(url) = mouse_state {
-                let url_bounds = url.linear_bounds(self.ctx.terminal());
-                self.ctx.terminal_mut().set_url_highlight(url_bounds);
+        } else if button == MouseButton::Left {
```

Comment:
I usually try not to use `if let` in this way to work around missing `&&` support, but if this is less than 100 columns it might be fine to change. It's just generally a bit hacky I think.

I've thought about relying on something like `matches!` for this, though I'm generally not a fan of pulling in depedencies for this kinda stuff. But it allows much nicer code imho:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0e057bc038b7a8f1d557a80003bbf190

Reply:
it shouldn't fit 100 columns, so its fine to ignore.